server/itx/server.go

```diff
@@ -92,6 +94,10 @@ func newServer(cfg config.Config, testing bool) (*Server, error) {
 
 // Start starts the server
 func (s *Server) Start(ctx context.Context) error {
+	if err := s.TurnOn(); err != nil {
+		return errors.Wrapf(err, "failed to turn on itx server")
+	}
```

Comment:
if this function is called at beginning, what's the definition of `s.Ready()`?

Reply:
it is `s.Ready()`, feel `TurnOn/Off` is more clear (action verb instead of Ready/NotReady() can be interpreted as status)